ext/debug/segger/Kconfig

```diff
@@ -8,9 +8,53 @@ config HAS_SEGGER_RTT
 	bool
 	default n
 
+if HAS_SEGGER_RTT
+
 config SEGGER_SYSTEMVIEW
 	bool
 	prompt "Segger SystemView support"
 	default n
-	depends on HAS_SEGGER_RTT
 	select RTT_CONSOLE
+
+config SEGGER_RTT_MAX_NUM_UP_BUFFERS
+	int "Maximum number of up-buffers"
+	default 3
+
+config SEGGER_RTT_MAX_NUM_DOWN_BUFFERS
+	int "Maximum number of down-buffers"
+	default 3
+
+config SEGGER_RTT_BUFFER_SIZE_UP
+	int "Size of the buffer for terminal output of target, up to host"
+	default 1024
+
+config SEGGER_RTT_BUFFER_SIZE_DOWN
+	int "Size of the buffer for terminal input of target, from host"
+	default 16
+
+config SEGGER_RTT_PRINTF_BUFFER_SIZE
+	int "Size of buffer for RTT printf to bulk-send chars via RTT"
+	default 64
+
+choice
+	prompt "Mode for pre-initialized terminal channel (buffer 0)"
+	default SEGGER_RTT_MODE_NO_BLOCK_SKIP
+
+config SEGGER_RTT_MODE_NO_BLOCK_SKIP
+	bool "Skip. Do not block, output nothing."
+
+config SEGGER_RTT_MODE_NO_BLOCK_TRIM
+	bool "Trim: Do not block, output as much as fits."
+
+config SEGGER_RTT_MODE_BLOCK_IF_FIFO_FULL
```

Comment:
Do we still need to configure this since you've added the `wait()` code in `rtt_console.c`?

Reply:
for consistency i think it's worth moving full configuration to kconfig.